### DISCUSSION TITLE
Getting error when I run the new Tina starter command.

### DISCUSSION THREAD

**Discussion by rafidalhaque** [tinacms/tinacms#2483] [Help]
**Hello!**
When I run this :arrow_down:  command to setup a new Tina starter from CLI,

```bash
npx create-tina-app@latest
```

I receive this error message,
```bash 
npm ERR! code E404
npm ERR! 404 Not Found - GET https://packagecloud.io/github/git-lfs/npm/isexe/-/isexe-2.0.0.tgz
npm ERR! 404 
npm ERR! 404  'isexe@https://packagecloud.io/github/git-lfs/npm/isexe/-/isexe-2.0.0.tgz' is not in this registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     ~/.npm/_logs/2022-01-25T06_30_58_841Z-debug-0.log
```

How can I fix it, please? 

**P.S.:** Log is attached. 

**Log:**
[2022-01-25T06_30_58_841Z-debug-0.log](https://github.com/tinacms/tinacms/files/7931200/2022-01-25T06_30_58_841Z-debug-0.log)

**Comment by perkinsjr** [comment]
Hey there, 

Can you try again? I just attempted this on two systems and both created new Tina applications without any issues. 

If it continues to persist can you provide:

1. OS
2. node + NPM versions

Cheers!

**Reply by rafidalhaque** [reply]
> 

Hi @perkinsjr! 
Thanks for your response. I tried it again & it shows me the same error, my - 

**System Info:**


**OS:** Manjaro Linux x86_64 
**Kernel:** Linux 5.15.16-1-MANJARO 
**Shell:** bash 5.1.16 
**DE:** Plasma 5.23.5  
**node:** v16.13.0
**npm:** 8.3.0

**Reply by perkinsjr** [reply]
This seems to be related to something that is outside of Tina's control and not related to your packages:

can you try

```
sudo apt-get update
```

Then

```
npm clean cache
```

Then try `npx-create-tina-app@latest`

If that doesn't work can you confirm that `npx create-next-app@latest` works?

**Reply by rafidalhaque** [reply]
Sorry, none of the commands you suggest is working :disappointed:

**Reply by perkinsjr** [reply]
Can you provide any errors you get? This is a system issue versus a tina issue but we'd like to try to help if possible.
